AppCenter/AppCenter/Internals/Storage/MSACDBStorage.m

```diff
@@ -101,7 +106,12 @@ - (int)executeQueryUsingBlock:(MSACDBStorageQueryBlock)callback {
   if (!db) {
     return result;
   }
-
+  if (self.pageSize <= 0) {
```

Comment:
Do we need this check in `setMaxStorageSize` as well?
Can we log a more specific message, like `MSACLogError([MSACAppCenter logTag], @"The database was not configured correctly. The page size is expected to be positive, actual value:%ld", self.pageSize)`?
Same for other similar logs.

Reply:
Fixed